packages/vm/src/instance.rs

```diff
@@ -393,7 +393,7 @@ mod singlepass_test {
 
         let init_used = orig_gas - instance.get_gas();
         println!("init used: {}", init_used);
-        assert_eq!(init_used, 45568);
+        assert_eq!(init_used, 65606);
```

Comment:
These are all quite huge gas increases for the safety check. I'm sure this can be optimized. But like I said, in a point release of serde-json-wasm, maybe in cosmwasm 0.8.1. This is an optimization that doesn't break any APIs